Content.Client/Chemistry/UI/ReagentCardControl.xaml.cs

```diff
@@ -0,0 +1,30 @@
+using Content.Shared.Chemistry;
+using Robust.Client.AutoGenerated;
+using Robust.Client.Graphics;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.XAML;
+
+namespace Content.Client.Chemistry.UI;
+
+[GenerateTypedNameReferences]
+public sealed partial class ReagentCardControl : Control
+{
+    public string StorageSlotId { get; }
+    public Action<string>? OnPressed;
+    public Action<string>? OnEjectButtonPressed;
+
+    public ReagentCardControl(ReagentInventoryItem item)
+    {
+        RobustXamlLoader.Load(this);
+
+        StorageSlotId = item.StorageSlotId;
+        ColorPanel.PanelOverride = new StyleBoxFlat { BackgroundColor = item.ReagentColor };
+        ReagentNameLabel.Text = item.ReagentLabel;
+        ReagentNameLabel.FontColorOverride = Color.White;
+        FillLabel.Text = item.StoredAmount;
+        EjectButtonIcon.Text = "⏏";
```

Comment:
Silly minor nit but this should be a loc string containing the symbol, not bare here. (Symbols like this can sometimes be culture specific. I don't think this one is, but it's good to play safe.)